### PR TITLE
Testing/User and UserPreferences modules

### LINKED ISSUE #141: Testing UserPreferences Module
## The story:
Adding unit tests to the UserPreferences Module.

---

## What we should do:
To get functional and branch percentages up, we need to make sure all of the resolver methods that can be tested are tested and as much branching (edge cases) covered as much as possible.

---

## To be successful we need to:
 - [x] Import the module in the utils/TestClient.ts file
 - [ ]  Create and run a test for each method of the service file in the tests/entities folder
 - [ ] Test every possible wrong scenario
 - [x] Ensure that each methods fullfill their API contract

### LINKED ISSUE #144: Testing User Module
## The story:
Adding unit tests to the User Module.

---

## What we should do:
To get functional and branch percentages up, we need to make sure all of the resolver methods that can be tested are tested and as much branching (edge cases) covered as much as possible.

---

## To be successful we need to:
 - [x] Import the module in the utils/TestClient.ts file
 - [x]  Create and run a test for each method of the service file in the tests/entities folder
 - [ ] Test every possible wrong scenario
 - [ ] Ensure that each methods fullfill their API contract

### PR DESCRIPTION
**Pull request by LeeYang2019**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Fixed issues (PRs without this will not be accepted)
- Fixes #141 
- Fixes #144 

## Changes
- Change 1: added more test cases to User Module

## PR Checklist
- [x] Does your PR contain relevant tests that pass?
- [x] Is your code linted? `yarn lint` or `yarn lint:fix`
- [x] Does your code build correctly in ALL projects? `yarn build`

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "eslint.format.enable": true,
+  "eslint.trace.server": "off",
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  }
+}
```
Comment (tristanMatthias) [inline .vscode/settings.json:7]: Remove this please :)

Comment (LeeYang2019) [reply .vscode/settings.json]: Hey @tristanMatthias, vscode config has been removed and also added to .getignore. I believe I've updated it appropriately. Thanks.

**Comment by tristanMatthias** [comment]
Hey @LeeYang2019! This looks good to merge to me. Was there more work required? Otherwise I'll merge it in :)

**Comment by LeeYang2019** [comment]
Hey @tristanMatthias, there's not more work. Please merge this in. Thanks.
